src/classes/BDI_Donations.cls

```diff
@@ -186,7 +186,7 @@ public class BDI_Donations {
         }
         
         // insert our new Opps
-        list<Database.SaveResult> listSR = database.insert(listOppInsert, false);
+        list<Database.SaveResult> listSR = UTIL_DMLService.insertRecords(listOppInsert, false);
```

Comment:
on line 201, we do database.update(listOppUpdate, false).  shouldn't this also go thru UTIL_DMLService?

Reply:
also in updatePayment() we do a database.update(listPmtUpdate, false).  do we care about this one?  I doubt users will put duplicate rules on payments, but who knows!  best to go thru UTIL_DMLService for consistency?

Reply:
Added the code for consistency. Good call.